proto/grp.proto

```diff
@@ -18,3 +20,22 @@ message Group {
   // True if this group has linked a Github token.
   bool github_linked = 4;
 }
+
+message CreateGroupRequest {
+  string name = 1;
+  string owned_domain = 2;
+  bool auto_populate_from_owned_domain = 3;
+}
+
```

Comment:
nit: can you copy / paste the examples above to these fields here so they all have examples?

Reply:
Done